rust/lance/src/dataset/scanner.rs

```diff
@@ -1259,7 +1277,7 @@ mod test {
                     ),
                     true,
                 ),
-                ArrowField::new("_distance", DataType::Float32, false),
+                ArrowField::new("_distance", DataType::Float32, true),
```

Comment:
Lets change these to `DIST_COL` as well?

Reply:
Okay. I've change all other instances or `ROW_ID` and `DIST_COL`.